components/clarity-lsp/src/common/requests/signature_help.rs

```diff
@@ -99,7 +97,12 @@ mod definitions_visitor_tests {
         source: &str,
         position: &Position,
     ) -> Option<Vec<lsp_types::SignatureInformation>> {
-        let contract = &ActiveContractData::new(Clarity2, Epoch21, None, source);
+        let contract = &ActiveContractData::new(
+            Clarity2,
+            clarity_repl::clarity::StacksEpochId::Epoch21,
```

Comment:
Curious why you decided to use a `use` for `Clarity2` but specify the fully qualified name for `Epoch21`?

Reply:
Valid point! Fixed